HighwayPatrol.cpp

```diff
@@ -1,10 +1,14 @@
 #include "HighwayPatrol.h"
 #include <iostream>
-#include <cassert>
+#include "Highway.h"
+#include "Vehicle.h"
+#include "Car.h"
+#include "SemiTruck.h"
+#include "Motorcycle.h"
+#include <typeinfo>
```

Comment:
Don't include what you don't use.
`<typeinfo>` is not needed or used in this source file
- [ ] revise